docs/guides/force-organizations.mdx

```diff
@@ -148,134 +200,116 @@ export function SyncActiveOrganization() {
+In Next.js App Router applications, you can use a [layout](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#layouts) to limit the entire application to only users with active organizations.
 
-<Callout type="info">
-  **How users can end up in this state ?**
-  - User just created an account, not by accepting an invitation, and hasn’t created an organization yet
-  - User has joined or created multiple organizations and leaves or deletes the active organization. 
-</Callout>
+In the example below, a component called `RequiredActiveOrgLayout` is created. This component will be used as a layout for all pages that require an active organization. If the user has an active organization, the children will be rendered. If the user does not have an active organization, the organization selection page will be rendered.
 
-Based on your usecase you can either decide to limit users in the entire app or a specific part of it. For example a B2B application might need to limit users with active organization for the entire app, whereas a B2B2C application would only limit the `/dashboard` path to users with active organization.
+```tsx filename="app/(with-active-organization)/layout.tsx"
+  import { OrganizationList } from "@clerk/nextjs"
+  import { auth } from "@clerk/nextjs/server"
+  import { PropsWithChildren } from "react"
 
-We can use the [`authMiddleware`](/docs/references/nextjs/auth-middleware) or we can use an `App Router Layout` to limit our entire application for users with an active organization.
+  export default function RequiredActiveOrgLayout(props: PropsWithChildren) {
+    // Get the organization ID from the session
+    const { orgId } = auth()
 
-<Tabs type="router" items={["App Router", "Pages Router"]}>
-    <Tab>
+    // If the user has an active organization, render the children
+    if (orgId) {
+      return props.children
+    }
 
-```tsx filename="app/(with-active-organization)/layout.tsx"
-import { OrganizationList } from "@clerk/nextjs"
-import { auth } from "@clerk/nextjs/server"
-import { PropsWithChildren } from "react"
+    // If the user does not have an active organization, render the organization selection page
+    return (
+      <section>
+        <h1>Welcome to the Organization Selection page.</h1>
+        <p>
+          This part of the application requires the user to select an
+          organization in order to proceed. If you are not part of an
+          organization, you can accept an invitation or create your own
+          organization.
+        </p>
+        <OrganizationList hidePersonal={true} />
+      </section>
+    )
+  }
+```
 
-export default function RequiredActiveOrgLayout(props: PropsWithChildren) {
-  const { orgId } = auth()
+In Next.js Pages Router applications, Clerk's [`authMiddleware`](/docs/references/nextjs/auth-middleware) helper can be used to limit the entire application to only user's with an active organization.
```

Comment:
`authMiddleware` can play the same role for App and Pages Router. Its agnostic in that sense. We should either do App Router as the only option (especially as near the top the guide says "This guide will be written for Next.js applications using App Router"), or provide App and Pages Router examples for /org-selection

@panteliselef -- I would contend this should all be switched to `clerkMiddleware` and we merge this next week when v5 releases. Thoughts?

Reply:
There are a lot of combos that can be used. The reasoning behind not having an snipper with middleware for App Router is that I see people wanted to do "page" or "layout" protection. For sure going with a middleware will be more performant. 

In a slack thread we discussed that we wanted to showcase a layout protection for app router and authMiddleware for pages router.

Regarding `clerkMiddleware` we can come back an update the guide

Reply:
I also thought about this and believe that because it can be used in both router-types, that we should showcase both. So we can change the text to be something like this:

```suggestion
Clerk's [`authMiddleware`](/docs/references/nextjs/auth-middleware) helper can be used to limit the entire application to only user's with an active organization. It can be used in both App Router and Pages Router applications.

... (show both App Router and Pages Router examples)

In Next.js App Router applications, instead of using `authMiddleware`, you also have the option to use a [layout](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#layouts) to limit the entire application to only users with active organizations.
```

Reply:
@panteliselef would you be able to throw together the App Router example for using `authMiddleware` ?

Reply:
This feels good to me.